#1 - Name, Role, Value (Loaded Data) - Based on Master

bae-junghwan opened the pull request:
Addresses Issue #1 AND Issue #72 . Previously, the buttons visually looked like links even though they were buttons. This PR amends that issue by making them visually buttons. The colors used should cover the 4.5:1 ratio requirement. In addition, the button labels shouldn't wrap until zoomed in a fair amount. Visuals are hard. 🥲 

![image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/123202460/1485a641-5f5d-4b30-9785-f18e1b16d209)
---
### Changes Made
**LoadedData.html**
- Buttons no longer use the `btn btn-link` class. This was what made them look like links. Replaced with `etl-button`

**styles.css**
- Added style to `etl-button` class elements. Colors fit the 4.5:1 accessibility ratio.

michellemcgowan reviewed (approved):
Nice job with this @bae-junghwan! I like that it is a button since it opens up a modal. When I zoomed in, it would wrap the text too and adjust which is really cool! All the functionality is still there as well. Ya did great.

Kelvyn-Meyers reviewed (changes requested):
Hi @bae-junghwan,

Good work getting these to look more like buttons! I talked to Natalie a bit and we think the wording on these buttons are a little long and the visuals could have some small touchups.

1. The button text is a bit long for each of these. Can you find a way to make them shorter without losing too much context? We also have the column header above to help identify the functionality of the button. We believe you can at least drop "Click to see" from the text.
2. The button's width is really long at higher resolutions due to your `width: 100%` value. If we're going to keep these buttons at a consistent width (regardless of the text), could you set the width of these buttons based on the longest-worded button text? This might be best figured out after completing the first request. I _think_ a hard-coded pixel width should be fine here, no need to do this programmatically.
3. DATIM usually has white text on a non-white background for buttons, so we should try to aim for that here, too. Could you set the text color for these buttons to white and then pick a blue background color that is color contrast compliant? ANDI should be able to recommend you a color on the "Color Contrast" tab. We also want the on-focus color to be compliant, too.

Feel free to experiment with other settings that you think would improve the visuals, too! Let me know if you have any questions.

bae-junghwan commented:
Hi @Kelvyn-Meyers ! Thank you for the review.

I just sent in a commit that hopefully addressed these issues. Instead of `Click to see evaluation included in State's ETL`, I relabeled it to `State ETL's Evaluation`. In addition, the button width is now a fixed 25 rem rather than 100%. Lastly, the colors used for the button are now blue background, white text.

Kelvyn-Meyers reviewed (changes requested):
Hi @bae-junghwan,

Thank you for getting those last changes in! Just a few more related (and unrelated) changes that just came up in another conversation:
1. The current button text has a typo for "Evaluationss". Also, can you replace "ETL's" with just "ETL"? I think "{state} ETL Evaluations" should be fine. If you disagree though, let me know and we can just stick with what you had.
2. Could you center the button text? We think the extra right-padding looks a little weird for buttons with less text, but we do like that you have a consistent button width. Centering the text might be the best option for these varying text lengths.
3. Is there anyway to line up the columns in Table 1 and Table 2? Natalie thinks it would look better that way, especially now that we have consistent widths applied to the buttons.
4. Natalie noticed the Table 1 and Table 2 headers are a little hard to identify as headers, especially Table 2's header. Could you experiment with ways to make it stand out more that looks similar to the general aesthetic of this website? If you increase the size at all, we just need to make sure all the other header sizes are adjusted to ensure visual header levels are easily identifiable. Adding some margin beneath each table could also help a bit, or experimenting with different colors and font-weights. Feel free to try different things here!

bae-junghwan commented:
Hello @Kelvyn-Meyers 

I've completed the requested changes.
- FIxed the typo and changed ETL's to ETL
- Button text is now centered
- Table 1 and Table 2's columns are now centered
- Changed Table headers to be more noticeable
